passkeys/templates/PassKeys.html

```diff
@@ -80,7 +80,7 @@
     function deleteKey(id,name)
     {
         $("#modal-title").html("Confirm Delete")
-        $("#modal-body").html("Are you sure you want to delete '"+name+"'? you may lose access to your system if this your only 2FA.");
+        $("#modal-body").text("Are you sure you want to delete '"+name+"'? you may lose access to your system if this your only 2FA.");
```

Comment:
That sounds strange (not fully sure about context though, I am new to this code).

Shouldn't that be something like "Are you sure you want to delete '"+name+"'? You may lose access to your account if this your only login method."?

- capitalised "You"
- system -> account ("your system" usually is not the same as "this server" / "your account on this server")
- you only lose access if you can't log in otherwise, not only by 2FA